public/app/features/explore/LogsContainer.tsx

```diff
@@ -36,7 +43,7 @@ export class LogsContainer extends PureComponent<LogsContainerProps> {
   getLogRowContext = async (row: LogRowModel, options?: any): Promise<any> => {
     const { datasourceInstance } = this.props;
 
-    if (datasourceInstance?.getLogRowContext) {
+    if (datasourceInstance && hasLogsContextSupport(datasourceInstance)) {
```

Comment:
```suggestion
    if (hasLogsContextSupport(datasourceInstance)) {
```

If you do a small change in the guard function to check if the `datasourceInstance` isn't undefined then you don't need to check that from call size like this. Will make the code read a bit better.

Reply:
Oh nice! I added it, thanks! 💪